app/services/scoring_service.py

```
@@ -0,0 +1,186 @@
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)